internal/fs/implicit_dirs_test.go

```diff
@@ -569,3 +573,109 @@ func (t *ImplicitDirsTest) AtimeCtimeAndMtime() {
 	ExpectThat(ctime, timeutil.TimeNear(mountTime, delta))
 	ExpectThat(mtime, timeutil.TimeNear(mountTime, delta))
 }
+
+func verifyInvalidPath(path string) {
+	_, err := os.Stat(path)
```

Comment:
lets add t.Helper() here for easy debugging

Reply:
done